src/__tests__/Price.Stale.test.ts

```diff
@@ -0,0 +1,40 @@
+import { AccountInfo, clusterApiUrl, Connection, PublicKey } from '@solana/web3.js'
+import { Magic, MAX_SLOT_DIFFERENCE, parsePriceData, Version } from '../index'
+
+test('Product', (done) => {
+  jest.setTimeout(60000)
+  const url = clusterApiUrl('devnet')
+  const ethPriceKey = 'EdVCmQ9FSPcVe5YySXDPCRmc8aDQLKJ9xvYBMZPie1Vw'
+  const connection = new Connection(url)
+  const publicKey = new PublicKey(ethPriceKey)
+
+
+  async function checkPriceStale() {
+    let accountInfo = await connection.getAccountInfo(publicKey)
```

Comment:
FYI, I think it would be better to download a binary blob of account data and run the test against that directly. It's not generally a good idea to hit live services (whose results can change) in unit tests.

I realize the other tests are hitting the live service, so it's ok with me if you continue doing so right now.

Reply:
Fixed.